File: test/lua/t-createtypedptr.lua
require 'utils'

local car = getCar()

local addr = car:f('name').address

-- Obtain a pointer to the name buffer.
--
local tp = dbgscript.createTypedPointer('kernelbase!char', addr)

-- Print the first few characters.
--
print (tp[0].value)
print (tp[1].value)

File: test/py/t-createtypedptr.py
from utils import *

car = get_car()

addr = car['name'].address

# Obtain a pointer to the name buffer.
#
tp = dbgscript.create_typed_pointer('kernelbase!char', addr)

# Print the first few characters.
#
print (tp[0].value)
print (tp[1].value)
